web-common/src/main/java/edu/unc/lib/boxc/web/common/services/XmlDocumentFilteringService.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.web.common.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import edu.unc.lib.boxc.model.api.xml.JDOMNamespaceUtil;
```

Comment:
Is this import in the right place?

Reply:
I think this one is in the right place according to my IDE